go.mod

```diff
@@ -1,17 +1,17 @@
 module github.com/bazelbuild/bazel-gazelle
 
-go 1.21
+go 1.21.1
 
 require (
-	github.com/bazelbuild/buildtools v0.0.0-20240313121412-66c605173954
+	github.com/bazelbuild/buildtools v0.0.0-20240827154017-dd10159baa91
```

Comment:
This update is what deleted so much in go.sum. The `golang.org/*` had no transitive changes other then what is this go.mod.